ietf-optical-impairment-topology.tree

```diff
@@ -49,39 +56,110 @@ module: ietf-optical-impairment-topology
                 +--:(concentratedloss)
                    +--ro concentratedloss
                       +--ro loss?   decimal64
-  augment /nw:networks/nw:network/nw:node/tet:te/tet:tunnel-termination-point:
+  augment /nw:networks/nw:network/nw:node/tet:te
+            /tet:tunnel-termination-point:
     +--ro OTSiG-element* [OTSiG-identifier]
     |  +--ro OTSiG-identifier    int16
     |  +--ro OTSiG-container
     |     +--ro OTSi* [OTSi-carrier-id]
     |        +--ro OTSi-carrier-id           int16
+    |        +--ro transponder-ref?          leafref
+    |        +--ro transceiver-ref?          leafref
+    |        +--ro configured-mode?          leafref
     |        +--ro OTSi-carrier-frequency?   decimal64
```

Comment:
I think it is the same as the `central-frequency` attribute below. As such, I would suggest removing it.

Reply:
The "central frequency" is related to the "transmit frequency" so the nominal central frequency configured for this transceiver, so it is the central frequency of frequency slot, while OTSi-carrier-frequency is the nominal central frequency of the modulated signal carried, so is the signal/carrier occupancy of the spectrum.

Reply:
Put OTSi-carrier-frequency in the common grouping . Delete central-frequency.  Change OTSi-carrier-frequency unit in THz. (defined in layer0-type-ext)